registration/include/pcl/registration/impl/default_convergence_criteria.hpp

```diff
@@ -59,7 +59,6 @@ pcl::registration::DefaultConvergenceCriteria<Scalar>::hasConverged ()
       convergence_state_ = CONVERGENCE_CRITERIA_ITERATIONS;
       return (true);
     }
```

Comment:
For readability reasons, it would be good to also remove the wrapping else {}. See https://clang.llvm.org/extra/clang-tidy/checks/readability-else-after-return.html